src/adapter/input/post_router.py

```
@@ -1,9 +1,10 @@
-from fastapi import APIRouter, Depends, status, HTTPException, Header
+from fastapi import APIRouter, Depends, status, HTTPException, Header, Request
 from core.usecase import PostUseCase
 from adapter.output.post_repository_impl import PostRepositoryImpl
 from infrastructure.sqlalchemy.config import SessionLocal
 from adapter.dto.post_dto import PostCreateRequest, PostCreateResponse, PostUpdateRequest, PostUpdateResponse, PostDetailResponse
```

Ruff (0.8.2)

`adapter.dto.post_dto.PostDetailResponse` imported but unused

Remove unused import: `adapter.dto.post_dto.PostDetailResponse`

(F401)